app/views/api/guidance/view-and-update-participant-data.md

```diff
@@ -0,0 +1,363 @@
+            "created_at": "2021-05-31T02:21:32.000Z",
+            "funded_place": true
+          }
+        ],
+        "participant_id_changes": [
+          {
+            "from_participant_id": "23dd8d66-e11f-4139-9001-86b4f9abcb02",
+            "to_participant_id": "ac3d1243-7308-4879-942a-c4a70ced400a",
+            "changed_at": "2021-05-31T02:22:32.000Z"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+## Retrieve a single participant’s data
+
+```
+GET /api/v3/participants/npq/{id}
+```
+
+### Check for participant ID changes
+
+Providers can check if an NPQ participant’s ID has changed using the `participant_id_changes` nested structure in the `NPQEnrolment`, which contains a `from_participant_id` and a `to_participant_id` string fields, as well a `changed_at` date value.
+
+### Duplicate IDs 
+
+We've [previously advised](https://manage-training-for-early-career-teachers.education.gov.uk/api-reference/release-notes.html#15-march-2023) of the possibility that participants may be registered as duplicates with multiple participant IDs. Where we identify duplicates, we'll fix the error by ‘retiring’ one of the participant IDs, then associating all records and data under the remaining ID.
+
+Where we identify duplicates, we'll fix the error by ‘retiring’ one of the participant IDs, then associating all records and data under the remaining ID. 
+
+To date, when this has occurred, we’ve informed providers of changes via CSVs
+
+For more detailed information, see the ['Retrieve a single participant' endpoint documentation](/api/docs/v3#/NPQ%20Participants/get_api_v3_participants_npq__id_).
+
+### Example response body
+
+```json
+{
+  "data": [
+    {
+      "id": "ac3d1243-7308-4879-942a-c4a70ced400a",
+      "type": "npq-participant",
+      "attributes": {
+        "full_name": "Isabelle MacDonald",
+        "teacher_reference_number": "1234567",
+        "updated_at": "2021-05-31T02:22:32.000Z",
+        "npq_enrolments": [
+          {
+            "email": "isabelle.macdonald2@some-school.example.com",
+            "course_identifier": "npq-senior-leadership",
+            "schedule_identifier": "npq-leadership-autumn",
+            "cohort": "2021",
+            "npq_application_id": "db3a7848-7308-4879-942a-c4a70ced400a",
+            "eligible_for_funding": true,
+            "funded_place": true,
+            "training_status": "active",
+            "school_urn": "123456",
+            "targeted_delivery_funding_eligibility": true,
+            "withdrawal": null
+            "deferral": null
+            "created_at": "2021-05-31T02:22:32.000Z"
+          }
+        ],
+        "participant_id_changes": [
+          {
+            "from_participant_id": "ac3d1243-7308-4879-942a-c4a70ced400a",
+            "to_participant_id": "23dd8d66-e11f-4139-9001-86b4f9abcb02",
+            "changed_at": "2023-09-23T02:22:32.000Z",
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+## Defer a particpant
```

Comment:
*participant

Reply:
thanks, all addressed in https://github.com/DFE-Digital/npq-registration/pull/1388/commits/714bd0867b375417e6b3724912d4e33e8e8f00d2